lua/gitblame/git.lua

```diff
@@ -38,12 +57,22 @@ local function get_repo_url(remote_url)
         return "https://" .. domain .. "/" .. path
     end
 
-    local url = string.match(remote_url, ".*git%@(.*)%.git")
+    local url = string.match(remote_url, ".*git@*ssh.dev.azure.com:v3/(.*)")
+    if url then
+        return get_azure_url(url)
+    end
+
+    local https_url = string.match(remote_url, ".*@dev.azure.com/(.*)")
```

Comment:
Can you give some examples of azure URLs? I'd like to make sure this will always work and be future proof. For example, will azure always use `v3` or might that version increment over time?

If you have any public Azure URLs that would be awesome too 😄 otherwise no worries!

Reply:
It is possible that the version will increment over time and the URLs will have to be composed differently. Maybe they'll even adhere to the same standards as other git providers (Clueless).

As to public repos, I don't really think there are any, however it seems you can create one for free here https://azure.microsoft.com/en-us/products/devops/repos

Reply:
Can we change this to use a variable version then? That way if they change from `v3` to `v4` it won't break this plugin suddenly

```suggestion
    local url = string.match(remote_url, ".*git@*ssh.dev.azure.com:v[0-9]/(.*)")
    if url then
        return get_azure_url(url)
    end

    local https_url = string.match(remote_url, ".*@dev.azure.com/(.*)")
```

Reply:
yup, good idea. Changed it :)